interfaces/builtin/netlink_audit.go

```diff
@@ -35,14 +35,22 @@ bind
 socket AF_NETLINK - NETLINK_AUDIT
 `
 
+const netlinkAuditConnectedPlugAppArmor = `
+# Description: Can use netlink to read/write to kernel audit system.
+network netlink,
+# CAP_NET_ADMIN required per 'man 7 netlink'
+capability net_admin,
```

Comment:
'man 7 netlink' only mentions that NET_ADMIN is needed for multicast sockets and I think this comment should be made clear wrt that. Eg:
```
# CAP_NET_ADMIN required for multicast netlink sockets per 'man 7 netlink'
```

In addition, 'man 7 capabilties' says that you will also need 'capability audit_read'. I suggest:
```
# CAP_AUDIT_READ required to read the audit log via the netlink multicast socket
# per 'man 7 capabilities'
capability audit_read,
```

Note that CAP_AUDIT_READ was only added in 3.16.

Finally, can you separate these with a blank line? Eg:
```
# Description: Can use netlink to read/write to kernel audit system.
network netlink,

# CAP_NET_ADMIN required for multicast netlink sockets per 'man 7 netlink'
capability net_admin,

# CAP_AUDIT_READ required to read the audit log via the netlink multicast socket
# per 'man 7 capabilities'
capability audit_read,
```